src/node_jsvmapi.h

```diff
@@ -31,20 +31,14 @@ NODE_EXTERN typedef void (*addon_abi_register_func)(
   void* priv);
 }  // namespace node
 
-struct napi_module_struct {
-  int nm_version;
```

Comment:
It may be prudent to add a napi version here, its easier to simply leave it at 1.0 than have to add an entry to the struct later on.

Thinking about it we may want to block off from private values at the end so that if we do need to expand later on we can do that as otherwise any change to this structure would for us into what we needed to do for 0.10.X with additional lookups.

I guess I'm suggested we add a version, and then say 4 void* fields at the end that we might use later on.